package.json

```diff
@@ -43,9 +46,10 @@
     "LICENSE"
   ],
   "devDependencies": {
+    "@sveltejs/vite-plugin-svelte": "^2.4.3",
     "@vitejs/plugin-react": "^3.1.0",
     "@vitejs/plugin-vue": "^4.1.0",
-    "chromedriver": "^112.0.0",
+    "chromedriver": "^114.0.3",
```

Comment:
we can remove it, since Nightwatch v3 handles driver management for us

Reply:
In this scenario, we must update the Nightwatch version. Should I proceed?

Reply:
Done 👍